common/test/data/scoreable/course.xml

```diff
@@ -0,0 +1 @@
+<course url_name="course" org="edx" course="jcd101"/>
```

Comment:
These files are auto-generated by course export.  I don't think there's much benefit in manually cleaning them up.  Maybe we could file a ticket to create the files more cleanly in the first place.